### PR TITLE
(Feat). Error 404 page

### PR DESCRIPTION
**Pull request by AVRahul**
Signed-off-by: A V RAHUL <a.v.rahul11@gmail.com>

Developed the Error 404 page, whenever use enters a wrong route this page will pop up.

Screenshot 2021-07-15 at 6 10 40 PM
Screenshot 2021-07-15 at 6 11 35 PM

### CODE REVIEW

File: website/public/locale/en/translation.yaml [hunk 1]
```diff
@@ -30,6 +30,12 @@ footer:
   copyrights: Copyright © 2021 The OpenEBS Authors | All rights reserved
   privacyPolicy: Privacy policy
 
+errorPage:
+  description: Page you're looking for was not found
```
Comment (satyapriyamishra222) [inline website/public/locale/en/translation.yaml:34]: oops! The page you're looking for was not found

File: website/public/locale/en/translation.yaml [hunk 2]
```diff
@@ -115,21 +120,21 @@ faq:
...
-    answer11: Kubernetes, of course, gives many ways to enable resilience. We leverage these wherever possible. For example, let’s say the IO container that has our iSCSI target fails. Well, it is spun back up by Kubernetes. The same applies to the underlying replica containers, where the data is stored. They are spun back up by Kubernetes. Now - the point of replicas is to make sure that when one or more of these replicas is being respun and then repopulated in the background by OpenEBS, the client applications still run. OpenEBS takes a simple approach to ensuring that multiple replicas can be accessed by an IO controller using a configurable quorum or the minimum number of replica requirements. Besides, our new cStor checks for silent data corruption and in some cases, can fix it in the background. Silent data corruption, unfortunately, can occur from poorly engineered hardware and from other underlying conditions, including those that your cloud provider is unlikely to report or identify.
+  question1: What is most distinctive about OpenEBS architecture?
```
Comment (satyapriyamishra222) [inline website/public/locale/en/translation.yaml:123]: Why these changes ?

File: website/src/pages/ErrorPage/index.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                            }}
+                        >
+                            {t("errorPage.homeBtnLable")}
+                        </Button>
+                    </Grid>
+                </Grid>
+                <img
+                    loading="lazy"
+                    width="100%"
+                    src="../images/svg/cloud_bottom.svg"
```
Comment (satyapriyamishra222) [inline website/src/pages/ErrorPage/index.tsx:48]: Does the name of the image justify its function here ?

File: website/public/locale/en/translation.yaml (hunk 2 above)
Comment (AVRahul) [reply website/public/locale/en/translation.yaml:123]: Did the format document whole file got formatted

File: website/src/pages/ErrorPage/index.tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        <div className={classes.root}>
+            <div className={classes.wrapper}>
+                <Grid
+                    container
+                    direction="row"
+                    justify="center"
+                    alignItems="center"
+                >
+                    <Grid item xs={12} lg={12}>
+                        <img
```
Comment (IsAmrish) [inline website/src/pages/ErrorPage/index.tsx:23]: The image tag should be  like ``.

File: website/src/pages/ErrorPage/index.tsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+                            className={classes.solidButton}
+                            onClick={() => {
+                                history.push("/");
+                            }}
+                        >
+                            {t("errorPage.homeBtnLable")}
+                        </Button>
+                    </Grid>
+                </Grid>
+                <img
```
Comment (IsAmrish) [inline website/src/pages/ErrorPage/index.tsx:45]: The image tag should be  like ``

File: website/src/pages/ErrorPage/index.tsx (hunk 4 above)
Comment (AVRahul) [reply website/src/pages/ErrorPage/index.tsx:23]: Updated it.

File: website/src/pages/ErrorPage/index.tsx (hunk 5 above)
Comment (AVRahul) [reply website/src/pages/ErrorPage/index.tsx:45]: Updated it.

File: website/public/locale/en/translation.yaml [hunk 6]
```diff
@@ -89,7 +94,7 @@ newsletter:
 joinCommunity:
   title: Join our friendly community
   slackAlt: Join on Slack
-  slackTitle: 'Get invitation to join #OpenEBS on Kubernetes Slack'
+  slackTitle: "Get invitation to join #OpenEBS on Kubernetes Slack"
```
Comment (satyapriyamishra222) [inline website/public/locale/en/translation.yaml:97]: Why quote for this one particular string ?

**Review by satyapriyamishra222 — APPROVED** [review]

**Review by IsAmrish — APPROVED** [review]

File: website/src/pages/ErrorPage/index.tsx [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+    return (
+        <div className={classes.root}>
+            <div className={classes.wrapper}>
+                <Grid
+                    container
+                    direction="row"
+                    justify="center"
+                    alignItems="center"
+                >
+                    <Grid item xs={12} lg={12}>
```
Comment (Pallavi-PH) [inline website/src/pages/ErrorPage/index.tsx:22]:
```suggestion
                    <Grid item xs={12}>
```
lg={12} is not required. xs={12} will take 12 for the devices above it i.e., md and lg

**Review by rakeshPRaghu — APPROVED** [review]

File: website/public/locale/en/translation.yaml (hunk 6 above)
Comment (AVRahul) [reply website/public/locale/en/translation.yaml:97]: These changes are not done now, it changed when the document was formatted. Though, the reason for quotes for this string was it has a special character `#` in order to handle that quotes were required.

Comment (AVRahul) [reply website/public/locale/en/translation.yaml:97]: cc @Pallavi-PH

Comment (Pallavi-PH) [reply website/public/locale/en/translation.yaml:97]: Yes, in order to include special characters as string in translation we need to wrap in in quotes

**Review by Pallavi-PH — APPROVED** [review]
